Tools/CISupport/delete-stale-build-files

```diff
@@ -86,6 +91,21 @@ def webkitBuildDirectory(platform, fullPlatform, configuration):
     return subprocess.Popen(['perl', os.path.join(os.path.dirname(__file__), "..", "Scripts", "webkit-build-directory"),
         "--" + platform, "--" + configuration, '--top-level'], stdout=subprocess.PIPE).communicate()[0].strip()
 
+def deleteWindowsStaleFiles():
+    directoriesToDelete = ['/tmp',
```

Comment:
Blindly deleting `/tmp` is not a good idea.

Reply:
Does `/tmp` exist on Windows? Would temp not be something like `/cygdrive/c/buildbot/AppData/local/Temp`? Given there is no "root" (rather only drive mount points) on NTFS.